File: tournaments/splitAddress/splitAddress.cs
string[] splitAddress(string address) {
    var result = new List<string>();
    var tokens = address.Split('/');

    result.Add(tokens[0].Substring(0, tokens[0].Length - 1));
    result.Add(tokens[2].Split('.')[0]);

    if (3 < tokens.Length) {
        result.Add(tokens[3]);
    }

    return result.ToArray();
}

File: tournaments/splitAddress/splitAddress.js
function splitAddress(address) {
    const result = [];
    const tokens = address.split('/');

    result.push(tokens[0].substring(0, tokens[0].length - 1));
    result.push(tokens[2].split('.')[0]);

    if (tokens[3]) {
        result.push(tokens[3]);
    }

    return result;
}
